### DISCUSSION TITLE
posts/2023/10/big-rebundle/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#35] [Announcements]
# posts/2023/10/big-rebundle/

Aided by an AI-generated script, I undo my hasty action from earlier this year.

https://www.brycewray.com/posts/2023/10/big-rebundle/?utm_campaign=RSS&utm_source=RSS&utm_medium=RSS

**Comment by idarek** [comment]
Thank you :)

**Reply by brycewray** [reply]
@idarek YW, sir. (And thanks for the plug you gave this post on the Hugo Discourse.)
